From 3d72cf67e2874bb78c8f871378244103933ba59b Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Mon, 2 Apr 2018 19:29:39 +0300
Subject: [PATCH 197/249] lineage: Sync NXP NFC stack

* Opt-in with TARGET_USES_NQ_NFC and building the right packages

Change-Id: I2a5a9419b8c6a7f8a9172c36319efe90f81519e9
---
 snippets/lineage.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 726d42e..8cc7319 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -103,6 +103,12 @@
   <project path="system/qcom" name="LineageOS/android_system_qcom" groups="qcom" />
   <project path="system/tools/dtbtool" name="LineageOS/android_system_tools_dtbtool" />
   <project path="vendor/codeaurora/telephony" name="LineageOS/android_vendor_codeaurora_telephony" />
+  <project path="vendor/nxp/interfaces/opensource/nfc" name="LineageOS/android_vendor_nxp_interfaces_opensource_nfc" >
+    <linkfile dest="vendor/nxp/interfaces/Android.bp" src="prop_pickup.bp"/>
+  </project>
+  <project path="vendor/nxp/opensource/external/libnfc-nci" name="LineageOS/android_vendor_nxp_opensource_external_libnfc-nci" />
+  <project path="vendor/nxp/opensource/frameworks" name="LineageOS/android_vendor_nxp_opensource_frameworks" />
+  <project path="vendor/nxp/opensource/packages/apps/Nfc" name="LineageOS/android_vendor_nxp_opensource_packages_apps_Nfc" />
   <project path="vendor/qcom/opensource/cryptfs_hw" name="LineageOS/android_vendor_qcom_opensource_cryptfs_hw" />
   <project path="vendor/qcom/opensource/dataservices" name="LineageOS/android_vendor_qcom_opensource_dataservices" />
   <project path="vendor/qcom/opensource/interfaces" name="LineageOS/android_vendor_qcom_opensource_interfaces" >
-- 
2.17.1

